talk/concurrency/threadlocal.tex

```diff
@@ -0,0 +1,22 @@
+\subsection[TLS]{Thread-local storage}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Thread-local storage}
+  \begin{block}{\mintinline{cpp}{thread_local} keyword}
+    \begin{itemize}
+      \item When declaring a variable thread-local, every thread will have its own copy
+      \item Most useful for things treated as ``working memory'' in each thread
+      \item Note: Can also be declared directly on the stack of a thread (faster)
+    \end{itemize}
+  \end{block}
+  \begin{exampleblock}{}
+    \begin{cppcode*}{}
+      thread_local int a{0};
+      std::thread t1([&](){ a++; });
+      std::thread t2([&](){ a++; });
```

Comment:
might also want to use a `jthread` instead 
```suggestion
      std::thread t1([&] { a++; });
      std::thread t2([&] { a++; });
```

Reply:
You are generally right, but the example is supposed to remind students of other examples which we are showing. There, we don't use `jthread` yet.
On second thought, though, I'll accept the suggestion, since this gives us the opportunity to show it in action.